client/extensions/wp-super-cache/miscellaneous.jsx

```diff
@@ -63,6 +63,14 @@ const Miscellaneous = ( {
 							</span>
 						</FormToggle>
 						}
+						{ ! compressionDisabledMessage &&
+						<Notice
+							isCompact
+							className="wp-super-cache__toggle-notice"
+							text={ translate( 'Compression is disabled by default because some hosts have problems ' +
+								'with compressed files. Switching it on and off clears the cache.' ) }
+							/>
+						}
```

Comment:
Yeah, it's a bit confusing. Basically, this new notice is always visible when its corresponding radio button is visible:

![persistent-compression-disabled-notice](https://user-images.githubusercontent.com/1190420/26876863-fc935a4a-4b55-11e7-912f-3dfba9dbc69e.jpg)

However, there's another notice returned by the API that disables compression completely and hides the radio button (and its corresponding notice) altogether:

![compression-disabled-notice](https://user-images.githubusercontent.com/1190420/26876892-1908e35c-4b56-11e7-9d5c-fc2672591ab8.jpg)